packages/components-react/src/heading/Heading.tsx

```diff
@@ -0,0 +1,47 @@
+import { Heading as UtrechtHeading } from '@utrecht/component-library-react/dist/css-module';
+import { HTMLAttributes, PropsWithChildren, ReactNode } from 'react';
+
+type HeadingLevels = 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface LuxHeadingProps extends HTMLAttributes<HTMLHeadingElement> {
+  appearance?: HeadingLevels;
+  level: HeadingLevels;
+}
+
+export const LuxHeading = ({
+  children,
+  level = 6,
+  appearance = level,
+  ...props
+}: PropsWithChildren<LuxHeadingProps>): ReactNode => {
+  const _level = level > 6 || level < 1 ? 6 : level;
+  const _appearance = appearance > 6 || appearance < 1 ? 6 : appearance;
+  return (
+    <UtrechtHeading
+      level={_level}
+      {...(_appearance ? { appearance: `utrecht-heading-${_appearance}` } : {})}
```

Comment:
I think it would be better to create an `Array` with these values. That would make this line more readable.

```suggestion
      // Somewhere else:
      const appearances = ['utrecht-level-1', ...];
      {appearances[_appearance]}
```